eng/testing/tests.mobile.targets

```diff
@@ -146,38 +146,37 @@
 
   </Target>
 
-  <UsingTask TaskName="WasmAppBuilder"
-             AssemblyFile="$(WasmAppBuilderTasksAssemblyPath)" />
+  <Import Project="$(MonoProjectRoot)\wasm\build\WasmApp.targets" Condition="'$(TargetOS)' == 'Browser'" />
+
+  <Target Condition="'$(TargetOS)' == 'Browser'" Name="BundleTestWasmApp" DependsOnTargets="WasmBuildApp" />
+
+  <Target Condition="'$(TargetOS)' == 'Browser'" Name="PrepareForWasmBuildApp" BeforeTargets="WasmBuildApp">
+    <PropertyGroup>
+      <WasmAppDir>$(BundleDir)</WasmAppDir>
+      <WasmMainAssemblyPath>$(PublishDir)WasmTestRunner.dll</WasmMainAssemblyPath>
+      <WasmMainJSPath>$(MonoProjectRoot)\wasm\runtime-test.js</WasmMainJSPath>
+      <WasmInvariantGlobalization>$(InvariantGlobalization)</WasmInvariantGlobalization>
+      <WasmGenerateRunV8Script>true</WasmGenerateRunV8Script>
```

Comment:
Is it certain that `'$(Scenario)' != 'WasmTestOnBrowser'` caused those Runtime tests failures? I saw them on https://github.com/dotnet/runtime/pull/45907 as well but wasn't certain because I was under the assumption that `run-v8.sh` was only used locally (not on CI) to help with development.

Reply:
Tests from `src/tests` seem to use it too, via `src/tests/Common/CLRTest.Execute.Bash.targets`, and `WasmTestRunner`.